tutorials/video_classification_example/train.py

```diff
@@ -179,6 +178,9 @@ class KineticsDataModule(pytorch_lightning.LightningDataModule):
     preprocessing transforms and configures the PyTorch DataLoaders.
     """
 
+    TRAIN_PATH = 'train.csv'
```

Comment:
Added these attrs so you can override w/ a different file/dir. In my case, I included `MiniKineticsDataModule` that has a slightly different directory structure.

Reply:
They aren't always files, though. The mini kinetics datamodule just uses dirs. Perhaps 'train_path_ext' or something?

Reply:
Ahh fair enough. Let's just stick with what you have then, TRAIN_PATH is good